templates/master/01-master-kubelet/baremetal/files/kubelet.yaml

```diff
@@ -0,0 +1,24 @@
+filesystem: "root"
```

Comment:
This also looks like the default one. This shouldn't be required.

Reply:
This may need changing - we have some questions about how the max pods was defined:  https://github.com/openshift/machine-config-operator/issues/723